nucleus/__init__.py

```diff
@@ -155,8 +188,7 @@ def delete_dataset(self, dataset_id: str) -> dict:
     def populate_dataset(
         self,
         dataset_id: str,
-        payload: dict,
-        local: bool = False,
+        dataset_items: List[DatasetItem],
         batch_size: int = 20,
```

Comment:
missing "force" parameter, which is important if someone wants to overwrite datasetItems

Reply:
Need to update doc string